joomla/joomla-cms
# Feature - Send Test Mail

This PR adds a new button ("Send Test Mail") in the Global Configuration with which the email settings can be tested with one click!

After the click on the button, the email data is transferred via an Ajax request to the server and a test mail is created and sent to the specified email address. If the sending process fails, a message with the error is displayed (see screenshots).

You don't have to save the configuration first, because the sending process will always be executed with the current entered values. Once the email settings are right, you should save them with a click on the "Save" button.

**How to test**

Install patch, go to the Global Configuration and click on the button! :-)
Change some settings (e.g. enter wrong values) and try it again.

Important: You should also test on an online server from which you really can send emails to receive the test mail. For instance, on local environments the "PHP Mail" option does not throw an error even though the email was not sent (if not configured properly)!

<img width="438" alt="bildschirmfoto 2015-12-10 um 10 16 56" src="https://cloud.githubusercontent.com/assets/1976103/11717284/c0b19036-9f4f-11e5-8f0d-ed012e814c73.png">

<img width="802" alt="bildschirmfoto 2015-12-10 um 14 56 33" src="https://cloud.githubusercontent.com/assets/1976103/11717078/5b078e30-9f4e-11e5-831d-dc568a2a543f.png">

<img width="802" alt="bildschirmfoto 2015-12-10 um 14 57 10" src="https://cloud.githubusercontent.com/assets/1976103/11717083/5f0f91b2-9f4e-11e5-9154-97d9595901fe.png">
